### PR TITLE
test: update e2e testcases

### PR DESCRIPTION
**Pull request by zdtsw**
## Description

- rename files
- add check on FTer CRD
- use label from labels.ODHAppPrefix
- update misleading log message
- re-order test cases
- rename DSC CR from e2e-test to e2e-test-dsc

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: tests/e2e/creation_test.go [hunk 1]
```diff
@@ -295,7 +305,7 @@ func (tc *testContext) testApplicationCreation(component components.ComponentInt
 	err := wait.PollUntilContextTimeout(tc.ctx, tc.resourceRetryInterval, tc.resourceCreationTimeout, false, func(ctx context.Context) (bool, error) {
 		// TODO: see if checking deployment is a good test, CF does not create deployment
 		appList, err := tc.kubeClient.AppsV1().Deployments(tc.applicationsNamespace).List(ctx, metav1.ListOptions{
-			LabelSelector: odhLabelPrefix + component.GetComponentName(),
+			LabelSelector: labels.ODHAppPrefix + component.GetComponentName(),
```
Comment (ykaliuta) [inline tests/e2e/creation_test.go:308]: it does not contain final /

Comment (ykaliuta) [reply tests/e2e/creation_test.go:308]: probably it's supposed to be used as labels.ODH.Component(component.GetComponenName())

File: tests/e2e/deletion_test.go [hunk 2]
```diff
@@ -71,7 +71,8 @@ func (tc *testContext) testApplicationDeletion(component components.ComponentInt
 
 	if err := wait.PollUntilContextTimeout(tc.ctx, tc.resourceRetryInterval, tc.resourceCreationTimeout, false, func(ctx context.Context) (bool, error) {
 		appList, err := tc.kubeClient.AppsV1().Deployments(tc.applicationsNamespace).List(ctx, metav1.ListOptions{
-			LabelSelector: labels.ODHAppPrefix + component.GetComponentName(),
+			//LabelSelector: labels.ODHAppPrefix + component.GetComponentName(),
+			LabelSelector: labels.ODH.Component(component.GetComponentName()),
```
Comment (ykaliuta) [inline tests/e2e/deletion_test.go:75]: 4 other places? Or it's just to check?

Comment (zdtsw) [reply tests/e2e/deletion_test.go:75]: pushed too fast............ not even finshed changes

Comment (ykaliuta) [reply tests/e2e/deletion_test.go:74]: > pushed too fast............ not even finshed changes

Ah, sorry. Been there :)

**Review by ykaliuta — APPROVED** [review]

**Pull request merged**